member/views.py

```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.serializers import ModelSerializer
+from rest_framework.views import APIView
+
+from member.models import Member
+
 
 # Create your views here.
+
+
+class MemberSerializer(ModelSerializer):
+    class Meta:
+        model = Member
+        fields = '__all__'
+
+
+class MemberListAPI(APIView):
+    def get(self, request):
+        person = Member.objects.all()
+        return Response(MemberSerializer(person, many=True).data)
+
+    def post(self, request):
+        # 튜플 형식 반환
+        person, is_exist_person = Member.objects.get_or_create(
+            name=request.data['name']
+        )
+        if is_exist_person:
+            return Response(MemberSerializer(person).data, status=201)
+        else:
+            return Response(status=409)
+
+
+class MemberDetailAPI(APIView):
+    def get(self, request, id_):
+        person = Member.objects.get(id=id_)
```

Comment:
```suggestion
        person = get_objects_or_404(Member, id=id_)
```
     이부분을 에러 핸들링해야해서  get_objects_or_404로 바꾸는게 좋을거같아여